composer.json

```diff
@@ -18,19 +18,19 @@
     "require": {
         "php": "^8.1",
         "ext-json": "*",
-        "auth0/auth0-php": "^8.3",
-        "auth0/login": "^7.4",
+        "auth0/auth0-php": "^8.5",
+        "auth0/login": "^7.5.1",
         "guzzlehttp/guzzle": "^6.0|^7.0",
-        "illuminate/contracts": "^6.0|^7.0|^8.0|^9.0",
-        "illuminate/support": "^6.0|^7.0|^8.0|^9.0",
-        "newrelic/monolog-enricher": "^2.0"
+        "illuminate/contracts": "^10.0",
+        "illuminate/support": "^10.0",
+        "marketredesign/newrelic-monolog-enricher": "dev-feature/monolog-v3"
```

Comment:
I accepted the PR for this, so you can probably update this with the new version.